### PR TITLE
Fix play kube command

### PR DESCRIPTION
**Pull request by haircommander**
Before, play kube wasn't properly setting the command. Fix this
Also, begin a dedicated test suite for play kube to catch regressions like this in the future

Begins to fix https://github.com/containers/libpod/issues/3286
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1729369

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

File: pkg/adapter/pods.go [hunk 1]
```diff
@@ -676,14 +676,15 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
...
-	if len(containerConfig.Command) != 0 {
+	if len(containerYAML.Command) != 0 {
 		containerConfig.Command = append(containerConfig.Command, containerYAML.Command...)
 	} else if imageData != nil && imageData.Config != nil {
 		containerConfig.Command = append(containerConfig.Command, imageData.Config.Cmd...)
 	}
 	if imageData != nil && len(containerConfig.Command) == 0 {
 		return nil, errors.Errorf("No command specified in container YAML or as CMD or ENTRYPOINT in this image for %s", containerConfig.Name)
 	}
+	fmt.Println(containerConfig.Command)
```
Comment (mheon) [inline pkg/adapter/pods.go]: Debugging?

Comment (haircommander) [reply pkg/adapter/pods.go]: oop, thanks

**Comment by haircommander** [comment]
happy green tests @rhatdan @vrothberg @TomSweeneyRedHat @QiWang19 @mheon PTAL

**Comment by mheon** [comment]
LGTM on my side. Like seeing integration tests.

**Comment by chenzhiwei** [comment]
@haircommander This fix was broken again in Podman v1.5.0.

**Comment by mheon** [comment]
How so? Can you file an issue?

**Pull request merged**